### DISCUSSION TITLE
更新日志 - 20230519

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#787] [Announcements]
# 系统：
1. 修复上一版本部分2.5G以上网卡协商速率问题（包括光模块）
2. UPNP 默认关闭 STUN 功能 #763

# 应用：
* QuickStart：
  1. 显示的网络接口按名称排序  #748
  3. 网络流量模块在小屏幕上自动减小高度
  4. 部分模块增加教程链接，页头增加 iStoreOS 官网链接
  5. 系统信息显示设备型号 （对于x86，机型数据是从bios读取出来的，如果bios没有机型数据，则会显示成“Default string”）

# 设备：
* x86/rk33xx/rpi4：集成部分 USB 无线网卡驱动，不常用的驱动可自行在软件包页面搜索安装 #781
* r2s：增加pwm风扇配置，50度启动，自动调速，暂无配置界面 #724

**Comment by jay5jiang** [comment]
R5S依然无法突破千兆。。。

**Reply by jjm2473** [reply]
你看看网口是不是协商到千兆以上了

**Reply by jay5jiang** [reply]
所有网口用的都是2.5G的，光猫连电脑拨号测速能到2000兆左右，用r5s的istore os拨号就是500兆，我问过很多人，就是istore os的问题。。。

**Reply by ayo5cuo** [reply]
我这也一样，istoreos下跑测速只能700左右，别的openwrt可以1000+

**Comment by ToutChenV587** [comment]
升级系统以后，点iStore发现维护有红点，就手贱升级了，然后就访问不了啦


404 未找到
对不起，请求的目标未找到。

No page is registered at '/admin/store'. If this url belongs to an extension, make sure it is properly installed. If the extension was recently installed, try removing the /tmp/luci-indexcache file.

无法调度: /cgi-bin/luci/admin/store/pages/maintance

**Reply by jjm2473** [reply]
参考这个重新安装试试 https://github.com/linkease/istore#%E5%AE%89%E8%A3%85-istore-%E5%88%B0-openwrt-%E5%AE%98%E6%96%B9%E5%9B%BA%E4%BB%B6

**Reply by ToutChenV587** [reply]
> 参考这个重新安装测试[https://github.com/linkease/istore#%E5%AE%89%E8%A3%85-istore-%E5%88%B0-openwrt-%E5%AE%98%E6 %96%B9%E5%9B%BA%E4%BB%B6](https://github.com/linkease/istore#%E5%AE%89%E8%A3%85-istore-%E5%88%B0-openwrt-%E5%AE%98%E6%96%B9%E5%9B%BA%E4%BB%B6)

哦，我知道啦，istore移到系统菜单下了，升级以后刷新原地址肯定就不能访问啦

**Reply by ToutChenV587** [reply]
有个新的问题，istore移到系统菜单下以后，在istore页面点击空白地方就会跳转到软件包的菜单

**Reply by jjm2473** [reply]
@ToutChenV587 官方的istore是在一级菜单的，你安装的不是官方版本。可能你增加了第三方的软件源，导致istore安装到了第三方版本。

**Reply by ToutChenV587** [reply]
> @ToutChenV587 官方的istore是在一级菜单的，你安装的不是官方版本。可能你增加了第三方的软件源，导致istore安装到了第三方版本。

可能是软件包定义过customfeeds，现在我注释掉了，选择了清华源。然后用系统便利工具，修复系统软件就一切恢复正常了
